internal/npm_install/generate_build_file.js

```diff
@@ -711,12 +711,29 @@ filegroup(
 
   if (pkg._executables) {
     for (const [name, path] of pkg._executables.entries()) {
+      // Handle additionalAttributes of format:
```

Comment:
seems like we also need to document this somewhere, maybe just start a wiki page about how to publish a package to NPM that works under Bazel?

Reply:
Was thinking that as well. Either a WIKI or add it to DEVELOPING.md and link from the main README.md.